kitsune/coolsearch/static/coolsearch/js/components/SearchForm.jsx

```diff
@@ -0,0 +1,86 @@
+/*
+React component that controls the behavior of the Search form.
+Handles the tabs of the form and the triggering of search queries.
+*/
+
+import SwipeViews from 'react-swipe-views';
+
+import SearchFormWiki from './SearchFormWiki.jsx';
+import SearchFormQuestion from './SearchFormQuestion.jsx';
+import SearchFormForum from './SearchFormForum.jsx';
+
+import QueryStore from '../stores/QueryStore.js';
+
+import QueryActionCreator from '../actions/QueryActionCreator.js';
+import SearchActionCreator from '../actions/SearchActionCreator.js';
+
+
+var getStateFromStores = function () {
```

Comment:
I think this should be a method of the `SearchForm` class instead of a separate function.